src/helpers/data.js

```diff
@@ -58,12 +45,13 @@ export default {
     if (!props.data) {
       return [];
     }
-    const accessor = Helpers.createAccessor(has(props, axis) ? props[axis] : axis);
-    const dataStrings = (props.data)
-        .map((datum) => accessor(datum))
-        .filter((datum) => typeof datum === "string");
-    // return a unique set of strings
-    return compact(uniq(dataStrings));
```

Comment:
yeah... but in this instance everything got pulled into a single loop rather than `map` then `filter` then `compact` then `uniq`. I could still pull out the `uniq`, but I need the reduce anyhow.